base/src/Data/Macaw/AbsDomain/AbsState.hs

```diff
@@ -611,6 +638,14 @@ asBoolConst (BoolConst b) = Just b
 asBoolConst TopV = Nothing
 
 -- | Add an integer to the abstract value.
+--
+-- This is not exported and is only used to increment or decrement the stack
+-- pointer by a known (by the architecture backend) signed integer after a call
+-- returns. The integer is *not* a bitvector.
```

Comment:
I'm a bit confused by the statement "The integer is *not* a bitvector", as the integer is passed directly to `addSignedOffsetBV` in some places, whose Haddocks emphatically state that the integer _is_ a bitvector. Which is it?

Reply:
Hmm this comment is referring specifically to the `Integer` argument to `bvinc`, which is indeed not a bitvector. It is only passed to `addSignedOffset` (the non-BV) version.

This is one of the few `Integer`s in the module that is not a bitvector, so I wanted to call it out (since I was initially worried about how to handle it while making this change, but then realized that it was the safe and easy case).

I can clarify that it is being called out because this is an exception to the normal order of business in the module